sssd_test_framework/topology.py

```diff
@@ -207,6 +207,16 @@ def test_ldap(client: Client, ldap: LDAP):
     .. topology-mark:: KnownTopology.IPATrustSamba
     """
 
+    Keycloak = SSSDTopologyMark(
+        name="keycloak",
+        topology=Topology(TopologyDomain("sssd", ipa=1, keycloak=1)),
+        domains=dict(test="sssd.keycloak[0]"),
+        fixtures=dict(ipa="sssd.ipa[0]", keycloak="sssd.keycloak[0]", provider="sssd.keycloak[0]"),
+    )
```

Comment:
I think that the provider should remain ipa and client should be part of this topology. This way we can use it to test IdP. Ideally, this pull request should allow us to test IdP (sssd -> ipa -> keycloak) but it may be difficult to implement authentication since it requires interaction with web browser. 

If you're not going to do it, then I think it is enough to just provide the keycloak host and role se we can built up on it eventually. In this case, you don't need to provide documentation and topology since we can not use it to test anything yet.

Reply:
Yes, I think I'll drop the topology for now.   It can be added later when adding the authentication part for the tests you describe.   I believe that should be coming early this year some time.

Reply:
dropped the topology for now so resolving this.